From c21eef3ff5c4bd38bc3b67f5209ccb839a725e77 Mon Sep 17 00:00:00 2001
From: Jacky Chou <jacky_chou@aspeedtech.com>
Date: Mon, 4 May 2026 18:16:15 +0800
Subject: [PATCH] dt-binding: reset: add AST2755 XPCS reset define

Rename SCU1 reset bit 43 from RESERVED to XPCS so AST2755
device trees can reference the XPCS reset line by name.

Upstream-Status: Backport [SDK v11.03 migration]
Signed-off-by: Jacky Chou <jacky_chou@aspeedtech.com>
Change-Id: I28546afa2e2ec4f5cd14cde441545ef9463fd0e5
Signed-off-by: rupeshs <rupeshs@ami.com>
---
 include/dt-bindings/reset/ast2700-reset.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/dt-bindings/reset/ast2700-reset.h b/include/dt-bindings/reset/ast2700-reset.h
index d272b452a4..860d66102f 100644
--- a/include/dt-bindings/reset/ast2700-reset.h
+++ b/include/dt-bindings/reset/ast2700-reset.h
@@ -97,7 +97,7 @@
 #define SCU1_RESET_I2C_FILTER	40
 #define SCU1_RESET_CALIPTRA	41
 #define SCU1_RESET_XDMA		42
-#define SCU1_RESET_RESERVED	43
+#define SCU1_RESET_XPCS		43	/* Added in AST2755 */
 #define SCU1_RESET_FSI		44
 #define SCU1_RESET_CAN		45
 #define SCU1_RESET_MCTP		46
-- 
2.43.0

